docs/source/gbd2017_models/causes/neoplasms/stomach_cancer/cancer_model.rst

```diff
@@ -0,0 +1,456 @@
+     - CSU stomach cancer cause-specific mortality rate forecast
+     - 2020-2040; defined as incidence cases per person-year in general population
+   * - disability_weight_s{248, 249, 250, 251}
+     - YLD appendix
+     - Sequela disability weights
+     - 0.288 (0.193-0.145), 0.049 (0.031-0.072), 0.451 (0.307-0.6), 0.54 (0.377-0.687)
+   * - prevalence_s{248, 249, 250, 251}
+     - GBD 2019, COMO, decomp_step='step4'
+     - stomach cancer sequelae prevalence
+     - Not forecasted
+   * - MST
+     - 4 months (95% CI: ); ?? distrbution of uncertainty at draw level
+     - Mean sojourn time; duration of time between onset of the asymptomtic stomach cancer to the clinical phase
+     - See below for instructions on how to sample and research background. NOTE: may update this value
+   * - AST
+     - ? (95% CI: ?); normal distribution of uncertainty at the draw level
+     - Average survival time; mean duration of time between detection and death
+     - See details below for sampling below. PLACEHOLDER VALUE
+
+.. todo::
+
+	Update/confirm placeholder values
+
+Mean Sojourn Time
+^^^^^^^^^^^^^^^^^
+
+**Parameter for Use in Model:**
+
+This parameter should be sampled *at the draw level* from the distribution detailed below and should be applied universally to all simulants within that draw.
+
+.. code-block:: Python
+
+  from scipy.stats import norm
+
+  # mean and 0.975-quantile of normal distribution for mean difference (MD)
+  mean = ??
+  q_975 = ??
+
+  # 0.975-quantile of standard normal distribution (=1.96, approximately)
+  q_975_stdnorm = norm().ppf(0.975)
+
+  std = (q_975 - mean) / q_975_stdnorm # std dev of normal distribution
+
+  # Frozen normal distribution for MST, representing uncertainty in the parameter
+  mst_distribution = norm(mean, std)
+
+.. note::
+
+  May consider adding individual-level variation to this parameter at a later date.
+
+
+Average Survival Time
+^^^^^^^^^^^^^^^^^^^^^
+
+**Parameter for Use in Model:**
+
+.. warning::
+
+  This is currently a stand-in value
+
+This parameter should be sampled *at the draw level* from the distribution detailed below and should be applied universally within that draw.
+
+.. code-block:: Python
+
+  from scipy.stats import norm
+
+  # mean and 0.975-quantile of normal distribution for mean difference (MD)
+  mean = ??
+  q_975 = ??
+
+  # 0.975-quantile of standard normal distribution (=1.96, approximately)
+  q_975_stdnorm = norm().ppf(0.975)
+
+  std = (q_975 - mean) / q_975_stdnorm # std dev of normal distribution
+
+  # Frozen normal distribution for AST, representing uncertainty in the parameter
+  ast_distribution = norm(mean, std)
+
+.. todo::
+
+  find values
+
+
+
+Validation Criteria
++++++++++++++++++++
+
+The incidence and prevalence of *clinical* stomach cancers in the general population should approximately validate to the GBD incidence and prevalence of stomach cancers. The mortality rates (CSMR and EMR) of stomach cancer should validate to those of GBD.
+
```

Comment:
Note that these will lag behind the GBD forecasts by the interval of MST

Reply:
noted!